### PR TITLE
Cleans up soapstone code

### PR DESCRIPTION
**Pull request by ghost**
Nope, I can't let the horse die. Now with 100% less fixed vocabularies.

* Renamed `already_message` to `existing_message` for clarity.
* Changed "the dull chisel is dull" message to "the dull chisel is too worn out to use".
* Removed the `our_message` var and replaced it with checks.
* Changed the inaccurate "you decide not to chisel anything" message displayed when someone else writes a message before you finish typing your own.
* Added spans to messages that lacked them.
* Changed the "finished chiseling" message into a visible one.
* Condensed `can_use()` into a single line.
* Made soapstone messages on a higher layer for visibility purposes.
* Cleaned up `locate()` checks to remove unnecessary type arguments.

### CODE REVIEW

File: code/modules/library/soapstone.dm [hunk 1]
```diff
@@ -44,75 +44,58 @@
...
 		return
-
 	playsound(loc, 'sound/items/gavel.ogg', 50, 1, -1)
-	user.visible_message("<span class='notice'>[user] starts [w_engraving] a message into [T].</span>", "You start [w_engraving] a message into [T].", "<span class='italics'>You hear a [w_chipping] sound.</span>")
-	if(can_use() && do_after(user, tool_speed, target=T) && can_use())
+	user.visible_message("<span class='notice'>[user] starts [w_engraving] a message into [T]...</span>", "<span class='notice'>You start [w_engraving] a message into [T]...</span>", "<span class='italics'>You hear a [w_chipping] sound.</span>")
+	if(can_use() && do_after(user, tool_speed, target = T) && can_use()) //This looks messy but it's actually really clever!
 		if(!locate(/obj/structure/chisel_message in T))
-			user << "You [w_engrave] a message into [T]."
+			user.visible_message("<span class='notice'>[user] leaves a message for future spacemen!</span>", "<span class='notice'>You [w_engrave] a message into [T]!</span>", "<span class='italics'>You hear a [w_chipping] sound.</span>")
```
Comment (coiax) [inline code/modules/library/soapstone.dm]: "future spacemen"

Really? :P

**Comment by coiax** [comment]
All of these are fine, I guess, but someone still needs to work out why persistence is broken.

File: code/modules/library/soapstone.dm (hunk 1 above)
Comment (Xhuis) [inline code/modules/library/soapstone.dm]: *Welcome gesture*

**Comment by Xhuis** [comment]
Didn't know that it was broken. I'll take a look later today.

**Pull request merged**